src/SolutionExplorerManagementApiRepository.ts

```diff
@@ -116,6 +138,47 @@ export class SolutionExplorerManagementApiRepository implements ISolutionExplore
     this.managementApi.deleteProcessDefinitionsByProcessModelId(this.identity, diagram.id);
   }
 
+  private async startPollingForDiagramChange(): Promise<void> {
+    this.isPolling = true;
+
+    const diagrams: Array<IDiagram> = await this.getDiagrams();
+
+    this.pollForDiagramChange(diagrams);
+  }
+
+  private async pollForDiagramChange(diagrams: Array<IDiagram>): Promise<void> {
+    setTimeout(async () => {
+      const newDiagrams: Array<IDiagram> = await this.getDiagrams();
```

Comment:
Manchmal wird ja schon Type Inference benutzt. 

Könnte man auch überall bei Gelegenheit mal nachziehen. 

Dann geht das hier z.B: so:

```suggestion
      const newDiagrams = await this.getDiagrams();
```